docs/doc/03-develop/01-python.md

```diff
@@ -37,75 +37,53 @@ Before you start, make sure you have successfully installed a local Databend. Fo
 
-This tutorial uses a SQL user named 'user1' with password 'abc123' as an example. As the program will write data into Databend, the user needs ALL privileges. For how to manage SQL users and their privileges, see https://databend.rs/doc/reference/sql/ddl/user.
+This tutorial uses a SQL user named 'user1' with password 'abc123' as an example. As the program will write data into Databend, the user needs ALL privileges. For how to manage SQL users and their privileges, see [User & Role](../14-sql-commands/00-ddl/30-user/index.md).
 
 ```sql
 CREATE USER user1 IDENTIFIED BY 'abc123';
 GRANT ALL on *.* TO user1;
 ```
 
-### Step 2. Configuring Connection String
-
-The driver supports various parameters that can be configured either as URL parameters or as properties passed to the Client. The two examples provided below demonstrate equivalent ways of setting these parameters for the common DSN:
-
-Example 1: Using URL parameters
-
-```python
-# Format: <schema>://<username>:<password>@<host_port>/<database>?<connection_params>
-client = Client.from_url('http://root@localhost:8000/db?secure=False&copy_purge=True&debug=True')
-```
-
-Example 2: Using Client parameters
-
-```python
-client = Client(
-    host='tenant--warehouse.ch.datafusecloud.com',
-    database="default",
-    user="user",
-    port="443",
-    password="password", settings={"copy_purge": True, "force": True})
-```
-
-To create a valid DSN, select appropriate connection parameters outlined [here](https://github.com/databendcloud/databend-py/blob/main/docs/connection.md) based on the your requirements.
-
-### Step 3. Write a Python Program
+### Step 2. Write a Python Program
 
 In this step, you'll create a simple Python program that communicates with Databend. The program will involve tasks such as creating a table, inserting data, and executing data queries.
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs groupId="python">
-<TabItem value="databend-py" label="databend-py">
+<TabItem value="databend-driver" label="databend-driver">
 
-You will use the databend-py library to create a client instance and execute SQL queries directly.
+You will use the databend-driver library to create a client instance and execute SQL queries directly.
 
-1. Install databend-py.
+1. Install databend-driver.
 
 ```shell
-pip install databend-py
+pip install databend-driver
 ```
 2. Copy and paste the following code to the file `main.py`:
 
 ```python title='main.py'
-from databend_py import Client
+from databend_driver import AsyncDatabendClient
+import asyncio
 
-# Setting secure=False means the client will connect to Databend using HTTP instead of HTTPS.
-client = Client('user1:abc123@127.0.0.1', port=8000, secure=False)
+async def main():
```

Comment:
The databend_driver currently only supports asynchronous mode. This aspect could be articulated more clearly in the documentation.